versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aci = {
       source  = "CiscoDevNet/aci"
-      version = ">= 2.6.1"
+      version = ">= 2.15.0"
```

Comment:
Is there a reason this change requires the minimum provider version to be upgraded? I agree it would be ideal to bump this version at some point, but probably under a separate change.

Reply:
The reason was to use the ACI provider version where the annotation changes have been incorporated. Can remove these version changes.